### ISSUE TITLE
False positive warning about import order

### ISSUE THREAD

**Issue by laike9m** [PyCQA/pylint#3222]
To reproduce 
```python
import bytecode
import deepdiff
```
`pylint` complains
```
test.py:3:0: C0411: third party import "import deepdiff" should be placed before "import bytecode" (wrong-import-order)
```

`bytecode` is a third-party library: https://pypi.org/project/bytecode/, but it seems pylint doesn't think so.

Python version: 3.7.4
Pylint version: 2.5.0.dev1

**Comment by PCManticore** [comment]
Thanks for the report! Can you do a `pip freeze` and check what version of `isort` you have installed? `pylint` relies on `isort` to power up these checks, so any issue with them might arise from a bug in `isort` itself.

**Comment by laike9m** [comment]
isort version is 4.3.21, aka the [latest version](https://github.com/timothycrosley/isort/releases).

**Comment by ralfstx** [comment]
I have the same problem with the `bottle` package. On

```py
import bottle
import requests
```
pylint complains:

> server.py:4:0: C0411: third party import "import requests" should be placed before "import bottle" (wrong-import-order)

When I replace `requests` with anything else, the error remains, so it seems that pylint deems `bottle` a *local* library.

bottle==0.12.18
requests==2.23.0

**Comment by DanielNoord** [comment]
@Pierre-Sassoulas I can't reproduce. Probably due to issues in `isort`. You might want to check for yourself and then possibly close this.

**Comment by Pierre-Sassoulas** [comment]
Thank you ! It's a problem with isort configuration imho (we're using isort internally), Might have to set ``known_third_parties`` to fix this. I'm going to close.
